docs/source/applications.rst

```diff
@@ -4,54 +4,25 @@
 Applications
 ************
 
-Introduction
-============
-
 Applications are python scripts built on the :ref:`Library` that execute a well defined task.
-These applications will be used as building blocks for the Simulation System Workflows.
-
-The application scripts can be found in gammasim-tools/applications.
+Applications are the building blocks of Simulation System Workflows.
+Application scripts can be found in gammasim-tools/applications.
 
-The applications expect a config file named config.yml in your gammasim-tools directory (unless
-specified differently using the function config.set_config_file_name) . Please, follow the instructions
-in :ref:`Configuration` to setup your config file. Alternatively, you can set up environmental
-variables through the *set_env_db_template.sh* script or pass it as arguments to the applications.
-The available arguments can be access by calling *--help* after calling the application.
+Each application is configured as described in :ref:`Configuration`.
+The available arguments can be access by calling the ``python application/<application name> --help``.
```

Comment:
the correct command would be ``python applications/<application name> --help``
's' missing

Reply:
Do you want to correct simple typos like this directly? This would be much faster!